packages/fxa-graphql-api/src/gql/lib/util.ts

```diff
@@ -0,0 +1,32 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { ILogger } from 'fxa-shared/log';
+
+// Temporary sanity check! Will be removed asap.
```

Comment:
We should probably file a ticket to:
* Confirm this is working as expected on stage
* Flip the env var on for React Signup prod if so
* Remove this code